services/api/emails/invite.md

```diff
@@ -0,0 +1,11 @@
+## Invite
+
+{{senderName}} ({{senderEmail}}) has invited you to join {{appName}}
+
+{{appName}} is a complete platform-as-a-service solution that enables unified ingestion and processing of data for the rapid and uniform development of applications.
+
+**[Accept Invite]({{{appUrl}}}/accept-invite?token={{token}})**
```

Comment:
my eyes blurred out looking at this but I think there's one too many `{` there no?

Reply:
oh I think I'm wrong this is for unescaping

Reply:
yeah i can also `{{&appUrl}}` perhaps thats more clear ?

Reply:
I think I'm more a fan of that syntax... is that even needed though? I thought that was HTML escaping

Reply:
also did you see my comment above about the single brace? What do you think?

Reply:
I am a bit on fence on this one ... like this is what https://mustache.github.io/ and others have landed on. 
I don't think its worth it, to go none default.   

 cc @dominiek

Reply:
I'm not in favor of following following default everything just to satisfy prettier (who can't even get newlines right ;) ) ... mustache also allows you to change the syntax for a reason too. I'd rather have templates that are easily readable.